### PR TITLE
Leaderboard View SwiftUI

### PR DESCRIPTION
**Pull request by barbarakos**
Added LeaderboardView.
Simulator Screen Shot - iPhone 14 Plus - 2022-12-15 at 11 49 00

### CODE REVIEW

File: QuizApp/Leaderboard/LeaderboardView.swift [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+                    .fontWeight(.light)
+                    .padding()
+
+            }
+
+            ScrollView {
+                ForEach(viewModel.leaderboard, id: \.index) { model in
+                    VStack {
+                        HStack {
+                            Text("\(model.index+1)")
```
Comment (marinzg) [inline QuizApp/Leaderboard/LeaderboardView.swift]: Space before and after `+`.

File: QuizApp/Quiz/QuizListViews/QuizListView.swift [hunk 2]
```diff
@@ -12,49 +12,52 @@ struct QuizListView: View {
...
-            VStack(alignment: .leading, spacing: 20) {
-                Picker("", selection: $segmentationSelection) {
-                    let items = ["All"] + CategorySection.allCases.map {$0.rawValue}
-                    ForEach(items, id: \.self) { option in
-                        Text(option)
-                    }
-                }
-                .onChange(of: segmentationSelection) { _ in
-                    getQuizzes()
+        VStack {
```
Comment (marinzg) [inline QuizApp/Quiz/QuizListViews/QuizListView.swift:14]: Will this be updated with my previous comment or is there some new logic here?

**Review by marinzg — CHANGES_REQUESTED** [review]

File: QuizApp/Quiz/QuizListViews/QuizListView.swift (hunk 2 above)
Comment (barbarakos) [reply QuizApp/Quiz/QuizListViews/QuizListView.swift:14]: It's updated with the previous comment.

**Comment by dorastipkovic** [comment]
Screenshot 2023-01-02 at 12 18 57
Tab bar is translucent.

File: QuizApp/Leaderboard/LeaderboardView.swift [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+                            Text("\(model.points)")
+                                .foregroundColor(.white)
+                                .font(.system(size: 25))
+                                .fontWeight(.heavy)
+                                .padding(.horizontal)
+                        }
+                        Divider()
+                            .overlay(.white)
+                            .frame(height: 3)
+                    }
```
Comment (dorastipkovic) [inline QuizApp/Leaderboard/LeaderboardView.swift]: Can  this be wrapped in a LazyVStack?

File: QuizApp/Quiz/QuizListViews/QuizListView.swift [hunk 4]
```diff
@@ -11,31 +11,33 @@ struct QuizListView: View {
...
-                                QuizCellView(quiz: quiz)
-                                    .onTapGesture {
-                                        viewModel.showQuizDetails(quiz: quiz)
-                                    }
+            }
+            .pickerStyle(SegmentedPickerStyle())
+            .cornerRadius(10)
+
+            ScrollView {
+                VStack(alignment: .leading, spacing: 20) {
```
Comment (dorastipkovic) [inline QuizApp/Quiz/QuizListViews/QuizListView.swift]: LazyVStack?

**Review by dorastipkovic — CHANGES_REQUESTED** [review]

**Review by dorastipkovic — APPROVED** [review]

**Pull request merged**